aten/src/ATen/xpu/detail/XPUHooks.cpp

```diff
@@ -0,0 +1,35 @@
+#include <ATen/xpu/XPUContext.h>
+#include <ATen/xpu/XPUDevice.h>
+#include <ATen/xpu/detail/XPUHooks.h>
+#include <c10/util/CallOnce.h>
+
+namespace at::xpu::detail {
+
+void XPUHooks::initXPU() const {
+  // TODO: the initialization of device allocator should be placed here.
+}
+
+bool XPUHooks::hasXPU() const {
+  return true;
+}
+
+std::string XPUHooks::showConfig() const {
+  return "XPU backend";
+}
+
+int XPUHooks::getGlobalIdxFromDevice(const at::Device& device) const {
```

Comment:
```suggestion
DeviceIndex XPUHooks::getGlobalIdxFromDevice(const at::Device& device) const {
```

Reply:
`device_id` in dlpack is `int` data type.